src/vi/truyenhentai18/src/eu/kanade/tachiyomi/extension/vi/truyenhentai18/TruyenHentai18.kt

```diff
@@ -37,13 +37,13 @@ class TruyenHentai18 : ParsedHttpSource() {
     // ============================== Popular ======================================
 
     override fun popularMangaRequest(page: Int) =
-        GET("$baseUrl/vi/truyen-de-xuat" + if (page > 1) "/page/$page" else "", headers)
+        GET("$baseUrl/truyen-de-xuat" + if (page > 1) "/page/$page" else "", headers)
```

Comment:
```suggestion
        GET("$baseUrl/$lang/truyen-de-xuat" + if (page > 1) "/page/$page" else "", headers)
```

Keep the `/vi/` part to avoid an extra redirection.